addons/website/static/tests/tours/snippet_popup_open_on_top.js

```diff
@@ -0,0 +1,40 @@
+/** @odoo-module */
+
+import tour from "web_tour.tour";
+import wTourUtils from "website.tour_utils";
+
+function fail (errorMessage) {
```

Comment:
Linter error: no space after function name

Reply:
this is removed in last version